napari/utils/misc.py

```diff
@@ -165,3 +165,16 @@ def __str__(self):
 
 
 callsignature = CallSignature.from_callable
+
+
+def all_subclasses(cls):
+    """Recursively find all subclasses of class ``cls``.
+
+    Returns
```

Comment:
For the sake of completeness, I think it would be nice to have a Parameters section, even though it's basically covered in the function description.

Reply:
ha! funnily-enough, my autoDocstring extension usually adds one, but must not have because of the `cls` variable name. 😅